### ISSUE TITLE
Container prefixes showing Network/Broadcast available of child prefix

### ISSUE THREAD

**Issue by enribla** [netbox-community/netbox#4744]
### Environment
* Python version: 3.6.8
* NetBox version: 2.8.5

### Steps to Reproduce
1. Create a prefix container. For example 10.181.90.0/24 
2. Create a child prefix. For example 10.181.90.40/29
![image](https://user-images.githubusercontent.com/53942090/84310805-897c7080-ab62-11ea-9a1a-7ce1b39e31ba.png)

3. Go to the parent prefix 10.181.90.0/24 and go to the tab ip addresses. The ips for the network and broadcast remain available and someone can use them
![image](https://user-images.githubusercontent.com/53942090/84310834-9600c900-ab62-11ea-988f-a204cbf6ae74.png)

### Expected Behavior
I expect that these ips appears Active with the information of the child prefix or if it-s a conainer it would be perfect if we cannot go direct to the ip addresses

### Observed Behavior
he ips for the network and broadcast of the child prefix remain available and someone can use them

This issue was also opened (https://github.com/netbox-community/netbox/issues/2436) but not fixed

**Comment by jeremystretch** [comment]
> Go to the parent prefix 10.181.90.0/24 and go to the tab ip addresses. The ips for the network and broadcast remain available and someone can use them

This is because, within the context of the prefix being viewed, they _are_ available. If you're using the prefix as a container for other prefixes, you'll need to consider available IPs within each child prefix, rather than from a parent prefix. The "child IP addresses" view has no real significance when viewing a container prefix. (Note that utilization for containers is calculated based on child prefixes, not IP addresses.) Perhaps we should simply hide the tab to avoid confusion.

**Comment by enribla** [comment]
Thanks Jeremy for your fast feedback
My point of view is that these ips are not available even from the parent prefix. As you tell: "The "child IP addresses" view has no real significance when viewing a container prefix" So he solution that you propose suits well to avoid mistakes and it looks coherent with the reality. Hiding the tab or giving RO Access from the container parent prefix should be fantastic. Many thanks in advance

**Comment by cpbruton** [comment]
This change really took me by surprise after I upgraded. Very frequently I want to list all IP addresses within a Container prefix. E.g. I have a /20 assigned to a site, with several /24s within it for individual networks. I have the /20 set as a 'Container' prefix and the /24s set as 'Active'. However I often want to see all of the IP addresses within the /20 in one list. Before this change, it was one click to do this. Now it's awkward - I have to go to the IP address search page and type or paste the parent prefix.

My workaround for now is changing my prefixes from 'Container' status to 'Active', but now I lose the utilisation information.

In short I don't feel like this was the correct fix for the OP's reported issue. There are many legitimate reasons one might want to list all IP addresses within a Container prefix.
